README.md

```diff
@@ -37,9 +37,19 @@ Wraps your component with the specified `Provider`, and sends the component's pr
 
         Example: `propsToContext(Provider, ({ propToIgnore, ...otherProps }) => otherProps)(Component)`
 
-### :zap: `contextToProps(Consumer)(BaseComponent)`
+### :zap: `contextToProps(Consumer, config?)(BaseComponent)`
 
-Wraps your component with the specified `Consumer`, spreading the context into its props.
+Wraps your component with the specified `Consumer`, and sends the context into its props. By default, the context will be spread into the component's props; you can optionally define a prop key for the context object, or a function to map to props manually.
+
+*   `Consumer`: a React context consumer, returned  by `React.createContext`
+*   `config`:
```

Comment:
Can you explicitly mark it as optional? (even if it is stated above, it's lost in the text a little bit, meaning it won't always be read)